sdk/compute/Azure.ResourceManager.Compute/src/Generated/Models/GallerySoftDeletedResource.cs

```diff
@@ -0,0 +1,87 @@
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="GallerySoftDeletedResource"/>. </summary>
+        /// <param name="location"> The location. </param>
+        public GallerySoftDeletedResource(AzureLocation location) : base(location)
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="GallerySoftDeletedResource"/>. </summary>
+        /// <param name="id"> The id. </param>
+        /// <param name="name"> The name. </param>
+        /// <param name="resourceType"> The resourceType. </param>
+        /// <param name="systemData"> The systemData. </param>
+        /// <param name="tags"> The tags. </param>
+        /// <param name="location"> The location. </param>
+        /// <param name="resourceArmId"> arm id of the soft-deleted resource. </param>
+        /// <param name="softDeletedArtifactType"> artifact type of the soft-deleted resource. </param>
+        /// <param name="softDeletedTime"> The timestamp for when the resource is soft-deleted. In dateTime offset format. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal GallerySoftDeletedResource(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, IDictionary<string, string> tags, AzureLocation location, ResourceIdentifier resourceArmId, SoftDeletedArtifactType? softDeletedArtifactType, string softDeletedTime, IDictionary<string, BinaryData> serializedAdditionalRawData) : base(id, name, resourceType, systemData, tags, location)
+        {
+            ResourceArmId = resourceArmId;
+            SoftDeletedArtifactType = softDeletedArtifactType;
+            SoftDeletedTime = softDeletedTime;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="GallerySoftDeletedResource"/> for deserialization. </summary>
+        internal GallerySoftDeletedResource()
+        {
+        }
+
+        /// <summary> arm id of the soft-deleted resource. </summary>
+        public ResourceIdentifier ResourceArmId { get; set; }
+        /// <summary> artifact type of the soft-deleted resource. </summary>
+        public SoftDeletedArtifactType? SoftDeletedArtifactType { get; set; }
+        /// <summary> The timestamp for when the resource is soft-deleted. In dateTime offset format. </summary>
+        public string SoftDeletedTime { get; set; }
```

Comment:
Could this be a type of `DateTimeOffset`? See an example [here](https://github.com/Azure/azure-sdk-for-net/blob/e4fce1607d65d709b19c81cc78969cd08b804244/sdk/hdinsight/Azure.ResourceManager.HDInsight/src/autorest.md?plain=1#L146).